debian/kodi-tools-texturepacker.install

```diff
@@ -1 +1 @@
-/usr/bin/TexturePacker
+/usr/bin/kodi-TexturePacker
```

Comment:
I have just realized that we are are not going to backport xbmc/xbmc#20413 into Matrix, while this change will affect both 19.x and 20.x. Maybe I should change this to `usr/bin/*TexturePacker` ?

Reply:
Pushed both directive files... this way `dh_install` will grab both `TexturePacker`, `kodi-TexturePacker`, `librrelec-TexturePacker` irrespectively to Kodi version.